contents/tutorials/ghost-analytics.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Quick start: Ghost analytics and session replay"
+date: 2024-10-11
+featuredTutorial: false
+author:
+  - danilo-campos
+tags:
+  - product analytics
+  - web analytics
+  - session replay
+---
+
+import Snippet from "../docs/integrate/snippet.mdx"
+
+With all the drama in the world of WordPress, it’s a great time to explore Ghost, a robust blogging and newsletter CMS. You can [self-host](https://ghost.org/docs/install/) Ghost on your own server, or lean on managed hosting from [Ghost.org](https://ghost.org).
+
+However you host, adding PostHog for [web analytics](/docs/web-analytics), [product analytics](/docs/product-analytics), and [session replay](/docs/session-replay) is a quick process that gives you immediate insight about how people are finding your site, enjoying your writing and exploring your archives. PostHog accounts are free, so sign up and follow along.
+
+## Adding PostHog using Ghost’s code injection
+
+Grab the PostHog snippet from [your project settings](https://us.posthog.com/project/2/settings/project#snippet):
+
+<Snippet />
+
+If you'd like to get person profiles for every visitor, update the value for `person_profiles` to `always`.
```

Comment:
This is sort of a confusing statement on its own. If I was coming here fresh, I would think "what's a person profile?" Unless it is necessary for the rest of the tutorial, I would probably remove it.

Maybe you can add a new section on identifying users once they sign up? Not necessary though.

Reply:
Fair enough! Complicated by that prophylactic iframe business, so I'll just nuke it